mmdeploy/backend/rknn/onnx2rknn.py

```diff
@@ -4,15 +4,28 @@
 import mmengine
 from rknn.api import RKNN
 
-from mmdeploy.utils import (get_common_config, get_onnx_config,
-                            get_partition_config, get_quantization_config,
+from mmdeploy.utils import (get_common_config, get_normalization,
+                            get_onnx_config, get_partition_config,
+                            get_quantization_config, get_rknn_quantization,
                             get_root_logger, load_config)
 from mmdeploy.utils.config_utils import get_backend_config
 
 
+def rknn_package_info():
+    """Get the rknn package information."""
+    import pkg_resources
+    toolkit = pkg_resources.working_set.by_key.get('rknn-toolkit', None)
+    toolkit = pkg_resources.working_set.by_key.get('rknn-toolkit2', toolkit)
+    if toolkit is None:
+        return dict(name=None, version=None)
+    else:
+        return dict(name=toolkit.project_name, version=toolkit.version)
+
+
 def onnx2rknn(onnx_model: str,
               output_path: str,
               deploy_cfg: Union[str, mmengine.Config],
+              model_cfg: Union[str, mmengine.Config],
```

Comment:
Backend should not rely on model_cfg.
This is just a `TODO NOTE`, we can refactor it in the future.

Reply:
The same here. We can just pass normalization.